### ISSUE TITLE
simple example is needed for Visual studio code 

### ISSUE THREAD

**Issue by EarlBread0** [wegi1/ESP32-8048S070-7INCH-LCD#1]
Dear friend,

I already integrated my code which is generated with squareline studio in LvglWidgets example.
I compliled my code with arduinoIDE but it takes too long to compile the code.
Therefore, I tried to compile my code in visual studio code environment. but it doesn't work.
I tested helloword example with visual studio code. It also doesn't work.

Is it possible to have simple example like helloworld for visual studio code?

Thanks in advance,

**Comment by wegi1** [comment]
Pleasse check github of my best friend Proteus:

https://github.com/dkm1978

which promissed put sth like that in his free time, so I think after a few days it could be share.
Exactly Proteus working the best with VSC. You can check below his example from VSC.

https://www.youtube.com/shorts/y_ZTh3PDInE
https://www.youtube.com/watch?v=wFTWB136DWo

https://www.youtube.com/watch?v=32n1yWHokdU
https://www.youtube.com/watch?v=O-QXzJEMmHQ

**Comment by EarlBread0** [comment]
Thanks for quick response. I visited this youtube and wrote comment there, and waiting for his reponse. He had example which called hello world with visual stuido code. That's what I want.
I appreciated again. Thanks

**Comment by dkm1978** [comment]
https://github.com/dkm1978/7.0inch_ESP32-8048S070_hello_world

**Comment by EarlBread0** [comment]
How a wondful you are! Thanks for your help. I solved my problem with Proteus. Happy thanksgiving~

**Comment by Rovel** [comment]
> https://github.com/dkm1978/7.0inch_ESP32-8048S070_hello_world

hello, this link is broken I would really appreciate to have also this vsc starter

**Comment by wegi1** [comment]
Yes, Proteus deleted this project bcos last one user didn't behavior of license agreement. So hi don't wanna be nice for every one when not everyone was fair.

**Comment by Rovel** [comment]
Fair @wegi1 I was able to use the Arduino examples, but as I try to build a brand new project using esp idf and adding https://github.com/lvgl/lv_port_esp32?tab=readme-ov-file#use-lvgl-in-your-esp-idf-project all goes bad

I was kind of going well in ESP IDF with the ESP32-2432S028 so I decided to get an 7inch display and newer esp chip.

But I cannot even build the project for esp32-s3, any guides on how to build a new project using esp idf (vscode extension), like default settings? 
- ESP IDF compatible versions?
- Do I choose de default esp32 or esp32-s3?  
- Also some guidance on the lvgl settings on menuconfig would be great too
